tests/contract-playground/src/InternalFunctions.sol

```diff
@@ -23,4 +24,9 @@ contract InternalFunctionExample {
         // This function is called only once within the contract.
         value = _newValue;
     }
+
+    function internalSet2(uint256 _newValue) internal {
+        // This function is called only once within the contract.
+        value = _newValue;
+    }
```

Comment:
Added this function.

Reply:
It doesn't start with `_` so we capture it. Now, there is another function called `_internalSet` - although it's used only once we don't capture it because the programmer has acknowledged it by starting the function name with an `_`